cowprotocol/cowswap
# Fix graphql generated files

# Summary

Follow up to https://github.com/cowprotocol/cowswap/pull/987

Adapting code dependent on generated file that's no longer generated at `src/state/data/generated.ts` from graphql data

- Hard coded ENS types
- Modded original UNI files which depended on it

ENS query still works!
![Screen Shot 2022-08-30 at 16 46 19](https://user-images.githubusercontent.com/43217/187483008-d536fa71-2d14-4b74-a117-d66217d5f3f3.png)


  # To Test

1. Check ENS address is loaded fine on affiliate page. Of course, requires address with ENS
You can use https://www.impersonator.xyz/ and use any ENS address to check it. 
Then just connect using WalletConnect

# Background

My previous fix should work as long as there’s a cached `src/state/data/generated.ts` file, but will break in fresh installs. This PR removes the need for that

Fresh install:
![Screen Shot 2022-08-31 at 11 38 43](https://user-images.githubusercontent.com/43217/187660221-41dbdea1-c87f-40ed-9de0-2f2102e53192.png)
